doc/tutorials/enable_s5.rst

```diff
@@ -160,21 +160,13 @@ The procedure for enabling S5 is specific to the particular OS:
 How to test
 ***********
 
-.. note:: The :ref:`CBC <IOC_virtualization_hld>` tools and service installed by
-   the `software-defined-cockpit
-   <https://github.com/clearlinux/clr-bundles/blob/master/bundles/software-defined-cockpit>`_ bundle
-   will conflict with the vUART and hence need to be masked.
+.. note:: Depends on the rootfs you used for SOS and UOS; system service maybe conflict with the vUART and hence need to be masked.
```

Comment:
It sounds like there are still technical issues and explanation needed about this change (what is the conflict, what are the consequences of masking the serial getty service, where do you put this systemctl command?) so it may be premature to talk about wording: 
```
Depending on the rootfs used for the Service and User VM, system services
might conflict with the vUART ``serial getty`` service.  Masking the ``serial getty`` service
will eliminate the conflict by stopping it from running even if there's a dependency on it.

For Yocto Project (Poky) or Ubuntu rootfs, mask the ``serial getty`` service using::

   systemctl mask serial-getty@ttyS1.service
```

Reply:
@dbkinder  Thank You!